client/task-list/tasks/payments/wcpay.js

```diff
@@ -75,35 +85,17 @@ class WCPay extends Component {
 		const { installStep } = this.props;
 		const { isPending } = this.state;
 
+		// When being redirected from the WCPay onboarding flow, don't render the Stepper so there isn't an extra "Plugins successfully activated" notice.
+		if ( getQuery()[ 'wcpay-connection-success' ] ) {
+			return null;
+		}
+
 		return (
 			<Stepper
 				isVertical
 				isPending={ ! installStep.isComplete || isPending }
-				currentStep={ installStep.isComplete ? 'connect' : 'install' }
-				steps={ [
-					installStep,
-					{
-						key: 'connect',
-						label: __(
-							'Verify business details',
-							'woocommerce-admin'
-						),
-						description: __(
-							'Verify your business details with our payment partner, Stripe.',
-							'woocommerce-admin'
-						),
-						content: (
-							<Button
-								isPrimary
-								isDefault
-								isBusy={ isPending }
-								onClick={ this.connect }
-							>
-								{ __( 'Verify details', 'woocommerce-admin' ) }
-							</Button>
-						),
-					},
-				] }
+				currentStep="install"
```

Comment:
Looks like the "Install WooCommerce Payments" spinner keeps spinning even after the notice appears ("Plugins were successfully installed and activated."), without any indication of what's happening. It would be nice – though probably not critical – to give UI feedback here of some sort.